pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "epidemics-suppression"
+version = "0.1.0"
+description = ""
+authors = ["Andrea Maiorana, Marco Meneghelli"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+matplotlib = "3.3.4"
+scipy = "1.6.1"
+
+[tool.poetry.dev-dependencies]
+black = "==19.10b0"
+isort = "*"
+pytest = "*"
```

Comment:
Pin these ones as well

Reply:
I'd pin with `^`, after all a reproducible install can always be obtained with the `poetry.lock`

Reply:
Done (I did the same for the other dependencies as well, except for Black)